### DISCUSSION TITLE
How to configure Ray in a CLI context?

### DISCUSSION THREAD

**Discussion by peterjaap** [spatie/ray#711] [Q&A]
I've tried creating ray.php in the same directory where I'm in, but it doesn't seem to work.

I composer required spatie/ray, created `test.php`, required the composer autoloader en put `ray('test.php');` in the file.

Then I ran `php test.php`, but no message shows up in Ray.

What am I missing?

**Comment by patinthehat** [comment]
@peterjaap Can you show your `test.php` file contents?

**Reply by peterjaap** [reply]
```
<?php

require_once 'vendor/autoload.php';

ray('test.php');
```

**Reply by patinthehat** [reply]
@peterjaap Could you also include your `ray.php` config file contents, as well as your OS, PHP version, etc.?  Please also check your settings in the Ray app and include the port it's listening on. Thanks! :pray:

**Reply by peterjaap** [reply]
I had this in my `ray.php`, according to [this blog](https://m.academy/blog/debug-magento-docker-spatie-ray/);

```
<?php
return [
    /*
     *  The host used to communicate with the Ray app.
     */
    'host' => 'host.docker.internal',
    /*
     *  The port number used to communicate with the Ray app.
     */
    'remote_path' => '/data/clientname/magento2',
    /*
     *  Absolute base path for your sites or projects on your local computer where your IDE or code editor is running on.
     */
    'local_path' => '/home/peterjaap/development/workspace/clientname/magento2',
];
```

I'm running Ubuntu 22.04, PHP 8.1. I'm using [this Docker dev env](https://github.com/JeroenBoersma/docker-compose-development). I left all options in the Ray app default, so it's running on port `23517`.
